internal/subshell/tcsh/tcsh.go

```diff
@@ -99,6 +94,15 @@ func (v *SubShell) RcFile() (string, error) {
 	return filepath.Join(homeDir, ".tcshrc"), nil
 }
 
+func (v *SubShell) EnsureRcFile() error {
+	rcFile, err := v.RcFile()
+	if err != nil {
+		return errs.Wrap(err, "Could not determine rc file")
+	}
+
+	return fileutils.TouchFileUnlessExists(rcFile)
+}
```

Comment:
Nit: Maybe put the logic for this in sscommon? ie. have this call something like `sscommon.EnsureRCFilExists(v)`.

Reply:
I tried that initially but none of these shell packages can import `sscommon` without an import cycle.